### PR TITLE
adding inputs from v1.0.1  to aorsf learner

### PR DESCRIPTION
**Pull request by bcjaeger**
As discussed in https://github.com/mlr-org/mlr3extralearners/pull/290#issuecomment-1764126879

### CODE REVIEW

File: tests/testthat/test_paramtest_aorsf_surv_aorsf.R [hunk 1]
```diff
@@ -7,6 +7,10 @@ test_that("paramtest surv.aorsf train", {
     "weights", # handled internally
     "oobag_fun", # handled internally
     "data", # handled internally
+    "n_thread", # handled internally
```
Comment (sebffischer) [inline tests/testthat/test_paramtest_aorsf_surv_aorsf.R:10]: The results of the parameter tests for the 'train' parameters are: 

Missing parameters:
- 'n_thread'
- 'sample_with_replacement'
- 'sample_fraction'
- 'split_rule'

This means they are exposed through the `aorfs::orsf()` function, so I don't understand why you add them to the `exclude` argument of the parameter test?

File: tests/testthat/test_paramtest_aorsf_surv_aorsf.R [hunk 2]
```diff
@@ -36,8 +40,11 @@ test_that("paramtest surv.aorsf predict", {
     "object", # handled internally
     "new_data", # handled internally
     "pred_horizon", # all times required for distr
+    "pred_aggregate", # handled internally
```
Comment (sebffischer) [inline tests/testthat/test_paramtest_aorsf_surv_aorsf.R:43]: Same as for the train parameters

File: R/learner_aorsf_surv_aorsf.R [hunk 3]
```diff
@@ -37,11 +37,17 @@ delayedAssign(
           n_tree = p_int(default = 500L, lower = 1L, tags = "train"),
           n_split = p_int(default = 5L, lower = 1L, tags = "train"),
           n_retry = p_int(default = 3L, lower = 0L, tags = "train"),
+          n_thread = p_int(default = 1, lower = 0, tags = "train"),
```
Comment (sebffischer) [inline R/learner_aorsf_surv_aorsf.R:40]: this misses the `"predict"` tag I think

**Review by sebffischer — CHANGES_REQUESTED** [review]
Thanks a lot for the PR. Please address my comments! :)

File: tests/testthat/test_paramtest_aorsf_surv_aorsf.R (hunk 2 above)
Comment (bcjaeger) [reply tests/testthat/test_paramtest_aorsf_surv_aorsf.R:43]: Same as above, will add a commit with this included.

File: tests/testthat/test_paramtest_aorsf_surv_aorsf.R (hunk 1 above)
Comment (sebffischer) [reply tests/testthat/test_paramtest_aorsf_surv_aorsf.R]: thanks!

**Comment by sebffischer** [comment]
Thanks a lot! :)

**Pull request merged**